tests/foreman/ui/test_notification_drawer.py

```diff
@@ -0,0 +1,229 @@
+# -*- coding: utf-8 -*-
+"""Test class for Notification Drawer
+
+:Requirement: Notification Drawer
+
+:CaseAutomation: notautomated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: UI
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from robottelo.decorators import stubbed, tier1, tier3, destructive
+from robottelo.test import UITestCase
+
+
+class NotificationDrawerTestCase(UITestCase):
+    """Implements Notification Drawer UI test cases"""
+
+    @stubbed()
+    @tier1
+    def test_positive_empty_notification_icon(self):
+        """Check empty notification icon is shown when there is no notification
+
+        :id: 6f5b7dfa-5607-4eee-b202-1954fefd1168
+
+        :steps: Clear all existing notifications
+
+        :expectedresults: Empty notification icon is shown
+
+        :CaseImportance: Critical
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_clear_notification(self):
+        """Check a notification is deleted by clicking on "Clear" button
+        within it
+
+        :id: 62e0694c-82c8-45a7-9918-cf867366ed22
+
+        :setup:
+            1. provision a new host
+
+        :steps:
+            1. Open Notification Drawer
+            2. Choose a notification
+            3. Click on "Clear" within it
+
+        :expectedresults: Assert chose notification is deleted from
+            Notification Drawer
+
+        :CaseImportance: Critical
+        """
+
+    @stubbed()
+    @destructive
+    def test_positive_notification_expiration(self):
+        """Check a notification expires after 24 hours
+
+        :id: 7fb71f80-cd85-4cb5-b498-92723f541238
+
+        :setup:
+            1. provision a new host
+            2. advance system clock more than 24 hours
```

Comment:
There is a `teardown` field for the list of available custom fields check the link below, in addition to those fields we have `title` and `description`. 

https://github.com/SatelliteQE/betelgeuse/blob/master/betelgeuse/__init__.py#L50-L80